src/popup/router/components/Modals/QrCodeReader.vue

```diff
@@ -175,20 +200,68 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@use "../../../../styles/variables";
-@use "../../../../styles/mixins";
+@use '../../../../styles/variables';
+@use '../../../../styles/mixins';
+@use '../../../../styles/typography';
 
 .qr-code-reader {
-  .icon {
-    color: variables.$color-blue;
+  .header {
```

Comment:
No need to put the code inside of this class - there is no need to have full HTML structure mirrored as the class structure.